newIDE/app/src/AssetStore/ResourceStore/ResourceCard.js

```diff
@@ -148,6 +158,7 @@ export const ResourceCard = ({ resource, onChoose, size }: Props) => {
           onChoose={onChoose}
           size={size}
           imageStyle={styles.previewIcon}
+          removeThemeFilterAppliedOnBackground // All svg are black and hard to see on background darken by the theme filter so we prefer using the raw checkered background that is quite light #nofilter
```

Comment:
I'm wondering if this is a bit too quick as a fix, and if we should rather fix the original problem which is: svgs should have a color that is the opposite of the theme, so that they're correctly visible in the store but also everywhere they're used.
(this change does not fix the fact that they're not visible elsewhere in dark theme)

Reply:
I think it would be quite a consequent job to do and could have impact on the performance.
I felt like having this as a fix to make sure they would be visible in the meantime was a good idea (only a few changes for a painpoint that is not a current priority).